roles/artifacts/tasks/environment.yml

```diff
@@ -19,6 +19,7 @@
     cmd: |-
       cp /etc/resolv.conf /etc/hosts {{ cifmw_artifacts_basedir }}/artifacts/
       cp -r /etc/NetworkManager/system-connections {{ cifmw_artifacts_basedir }}/artifacts/NetworkManager
+      cp -r /etc/yum.repos.d {{ cifmw_architect_basedir }}/artifacts/yum_repos
```

Comment:
As a side note, by default we generate the repositories in cifmw_basedir/artifacts/repositories:
https://github.com/openstack-k8s-operators/ci-framework/blob/main/roles/repo_setup/defaults/main.yml#L31

Having a distinct name is good imho.